src/gsy_e/gsy_e_core/rq_job_handler.py

```diff
@@ -84,12 +84,12 @@ def launch_simulation_from_rq_job(scenario, settings, events, aggregator_device_
         config = SimulationConfig(**config_settings)
 
         spot_market_type = settings.get("spot_market_type")
-        bid_offer_match_algo = settings.get("bid_offer_match_algo")
+        orders_match_algo = settings.get("orders_match_algo")
```

Comment:
The renaming of the `bid_offer_match_algo` into `orders_match_algo` was not reflected in the gsy-web.